beacon-chain/blockchain/testing/mock.go

```diff
@@ -30,6 +31,11 @@ func (ms *ChainService) ReceiveBlockNoPubsub(ctx context.Context, block *ethpb.B
 
 // ReceiveBlockNoPubsubForkchoice mocks ReceiveBlockNoPubsubForkchoice method in chain service.
 func (ms *ChainService) ReceiveBlockNoPubsubForkchoice(ctx context.Context, block *ethpb.BeaconBlock) error {
+	if ms.State == nil {
```

Comment:
Should we be receiving a block if we have no state ?

Reply:
This is a mock thing, i just wanted to minimize my test setup.

Reply:
Ah k, missed that it was a mock, ignore this comment them